tests/Unit/Persister/AsyncPagerPersisterTest.php

```diff
@@ -56,46 +56,4 @@ function ($message) {
         $pager = $this->createMock(PagerInterface::class);
         $sut->insert($pager);
     }
-
-    public function testInsertPageReturnObjectCount()
-    {
-        $persistersLocator = $this->createMock(ServiceLocator::class);
-        $persistersLocator->expects($this->once())->method('has')->with('foo')->willReturn(true);
-        $persistersLocator->expects($this->once())->method('get')->with('foo')->willReturn($this->createMock(ObjectPersisterInterface::class));
-
-        $pagerPersistersLocator = $this->createMock(ServiceLocator::class);
-        $pagerPersistersLocator->expects($this->once())->method('has')->with('in_place')->willReturn(true);
-        $pagerPersistersLocator->expects($this->once())->method('get')->with('in_place')->willReturn(
-            new InPlacePagerPersister(
-                new PersisterRegistry($persistersLocator),
-                $this->createMock(EventDispatcherInterface::class)
-            )
-        );
-
-        $pagerPersisterRegistry = new PagerPersisterRegistry($pagerPersistersLocator);
-
-        $pagerMock = $this->createMock(PagerInterface::class);
-        $pagerMock->expects($this->exactly(2))->method('setMaxPerPage')->with(10);
-        $pagerMock->method('setCurrentPage')->withConsecutive([1], [1], [0]);
-        $pagerMock->expects($this->exactly(2))->method('getCurrentPageResults')->willReturn([]);
-
-        $provider = $this->createMock(PagerProviderInterface::class);
-        $provider->expects($this->once())->method('provide')->with([
-            'first_page' => 1,
-            'last_page' => 1,
-            'indexName' => 'foo',
-            'max_per_page' => 10,
-        ])->willReturn($pagerMock);
-
-        $pagerProviderRegistry = $this->createMock(PagerProviderRegistry::class);
-        $pagerProviderRegistry->expects($this->once())->method('getProvider')->with('foo')->willReturn($provider);
-
-        $messageBus = $this->createMock(MessageBusInterface::class);
-        $sut = new AsyncPagerPersister($pagerPersisterRegistry, $pagerProviderRegistry, $messageBus);
-
-        $sut->insertPage(1, [
```

Comment:
funny fact this should have been assert result of `insertPage` but it's no doing it.